README.md

```diff
@@ -28,15 +28,15 @@ dist/omniSearch.js
 Run this to update files
 
 ```
-yarn
-yarn dev
+npm install
+npm run dev
```

Comment:
Consider running these commands in your `docker compose` setup:

```suggestion
docker compose run --rm php npm install
docker compose run --rm php npm run dev
```

Reply:
Changed it back, it seems to break docker to build outside of the folder: https://github.com/ITK-Leantime/leantime-omnisearch/blob/b9dc33db546d852bd91466df08aea95157c6cc47/webpack.config.js#L10

In dev mode it build directly into leantime...... I guess that is a bit hacky.

Reply:
It should always build into `./dist/js`, right?

Reply:
It probably should. When it builds, it builds into dist/js. when in "dev-mode", it builds directly into leantime.

Reply:
Yes, but we don't need that. When the plugin is installed, all its assets are symlinked into Leantime's `public` folder, so all changes in `./dist/js` during development will be reflected in the actual assets URL (after a browser “no cache” reload).